edx_exams/apps/core/models.py

```diff
@@ -159,6 +159,11 @@ class ExamAttempt(TimeStampedModel):
         ExamAttemptStatus.expired,
     ]
 
+    IN_PROGRESS_STATUSES = [
```

Comment:
I think this type of definition is better suited for https://github.com/edx/edx-exams/blob/main/edx_exams/apps/core/statuses.py, but great idea to make this accessible to other functions!